.github/workflows/test-runner-on-perf.yml

```diff
@@ -62,6 +63,74 @@ jobs:
               fi
             fi
           done
+      - name: "Fetch latest image git hash from data-repo dev"
+        id: "read_property"
+        run: |
+          CurrentVersion=$(curl -X GET "https://jade-perf.datarepo-perf.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.gitHash')
```

Comment:
Consider using snake uppercase for all the shell variables. This is usually the convention and makes them stand-out against the other commands.